examples/tutorials/analysis-3d/event_sampling_nrg_depend_models.py

```diff
@@ -262,7 +262,8 @@
 path = Path("$GAMMAPY_DATA/cta-caldb")
 irf_filename = "Prod5-South-20deg-AverageAz-14MSTs37SSTs.180000s-v0.1.fits.gz"
 
-pointing = SkyCoord(100.0, 30.0, frame="icrs", unit="deg")
+pointing_position = SkyCoord(100.0, 30.0, frame="icrs", unit="deg")
```

Comment:
ICRS is the default frame, I'd also highly recommend always using keyword arguments for coordinates, as it is not always obvious which order they are in (e.g. the argument order of `AltAz` coordinates is actually az, alt).

`SkyCoord(ra=100 * u.deg, dec=30 * u.deg)`